templates/node/src/client.ts.twig

```diff
@@ -271,6 +275,39 @@ class Client {
             data = await response.json();
         } else if (responseType === 'arrayBuffer') {
             data = await response.arrayBuffer();
+        } else if (response.headers.get('content-type')?.includes('multipart/form-data')) {
+            const body = await buffer(response.body);
+            const boundary = multipart.getBoundary(
+                response.headers.get("content-type") || ""
+            );
+            const parts = multipart.parse(body, boundary);
+            const partsObject: { [key: string]: any } = {};
+            
+            for (const part of parts) {
+                if (part.name) {
+                    if (part.name === "responseBody") {
+                        partsObject[part.name] = new Payload(part.data, part.filename || undefined);
+                    } else if (part.name === "responseStatusCode") {
+                        partsObject[part.name] = parseInt(part.data.toString(), 10);
+                    } else if (part.name === "duration") {
+                        partsObject[part.name] = parseFloat(part.data.toString());
+                    } else if (part.type === 'application/json') {
+                        try {
+                            let jsonData = JSON.parse(part.data.toString());
+                            if (Array.isArray(jsonData) && jsonData.length > 0 && 'name' in jsonData[0] && 'value' in jsonData[0]) {
+                                jsonData = Object.fromEntries(jsonData.map(item => [item.name, item.value]));
+                            }
+                            partsObject[part.name] = jsonData;
+                        } catch (e) {
+                            console.error('Error parsing JSON:', e);
```

Comment:
Woah, no. Just let the exception throw. Hiding it from developer with out custom logging logic is bad idea.